db/models/Events.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose')
+
+const categories = ['music', 'dining']
+const accessibilityOptions = ['braille-menu', 'wheelchair-friendly']
+
+const eventSchema = mongoose.Schema(
+  {
+    ownerId: { type: mongoose.Schema.Types.ObjectId, ref: 'User' },
+    name: { type: String, required: true },
+    category: { type: [String], enum: categories, required: true },
```

Comment:
we have a [String]- an array of strings. which apparently works (https://stackoverflow.com/questions/27447876/is-it-possible-to-create-a-multi-select-enum-in-mongoose) though we haven't tested it yet